### ISSUE TITLE
Feature Request: A search feature to select issuer/label

### ISSUE THREAD

**Issue by brian6932** [replydev/cotp#65]
I think a useful feature would be to be able to do a find search to select issuer/label with `ctrl + f` and/or `/`

Also one minor thing:
I think it would be nice if `k` were moved off this bind:
`k -> Show QRCode of the selected element`
and instead using `k`/`j` as vim alts for `↑`/`↓` would be better/more universal imo

**Comment by replydev** [comment]
Hi, i also think that a search feature could be useful, this commit d7b4a0cd1c116949705ec180a5c99beac05d942c includes a draft.
I want to improve the search UI, but for now let me know your impressions.

Thanks, regards.

**Comment by brian6932** [comment]
Nice, I like that `esc` lets me exit out of the find without exiting out of the whole program.
So my nitpicks would be:

1. I don't like that when you use find the whole screen gets taken up. If possible, I think it would be nice to put in a margin/overlay, maybe over the `cotp v0.2.0` text, or below the progress bar.

For example, this is how [bottom](https://github.com/ClementTsang/bottom/) does it:
https://i.nuuls.com/xgp3h.png

2. This is probably more annoying to do, but while using find, it should instead of only grabbing item on enter, track to closest item you're typing, with like a contains/similarity, and highlight them accordingly as you type. While only actually copying the item on enter. Since `esc` is already an exit for find, double enter seems unnecessary (as long as there is an overlay/margin like I mentioned in `1.`).

**Comment by replydev** [comment]
This 7702bd60c77ad4ac74e1f954907a860b8782f635 is quite better.

**Comment by brian6932** [comment]
Nice, much better. Now there's really only this nitpick:

> 2\. This is probably more annoying to do, but while using find, it should instead of only grabbing item on enter, track to closest item you're typing, with like a contains/similarity, and highlight them accordingly as you type. While only actually copying the item on enter. Since `esc` is already an exit for find, double enter seems unnecessary.

**Comment by replydev** [comment]
Yeah you are right. 0f8fa303043e4a35ec0156fa9bf6a24fea9d87c6 this could be the final step. Let me know.

Thank you, regards

**Comment by brian6932** [comment]
Perfect!
The only minor nitpick I could have is that the search box should support either, `ctrl + w` and/or `ctrl + backspace`, for deleting the whole word

**Comment by replydev** [comment]
> The only minor nitpick I could have is that the search box should support either, ctrl + w and/or ctrl + backspace, for deleting the whole word

Done with CTRL-W. Thank you for your advices.

**Comment by brian6932** [comment]
ctrl w/backspace is typically for single word deletion, but this is close enough, Ty for being so receptive ❤️
